sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_forecasting_v1/query.sql

```diff
@@ -0,0 +1,120 @@
+-- Query for contextual_services_derived.suggest_foreasting_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+-- DAU SHARE BY COUNTRY
+-- Used to scale automated global KPI forecasts down to US clients only
+WITH client_counts AS (
+  SELECT
+    country,
+    submission_date,
+    COUNT(*) AS total_clients,
+    COUNT(
+      CASE
+  -- SUGGEST DESKTOP ELIGIBILITY REQUIREMENTS
```

Comment:
These are the Suggest eligibility requirements: https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/suggest_clients_daily_v1/query.sql#L144

The 2 filters on version are to select versions >= 92.0.1